### DISCUSSION TITLE
Skólasetning reglunnar 2023 - 2023/7/2

### DISCUSSION THREAD

**Discussion by giscus** [Biskupsreglan/biskupssidan#33] [vefsida-ummæli]
# Skólasetning reglunnar 2023 - 2023/7/2

Völvan hafði kolrangt fyrir sér

https://www.biskupsreglan.com/frettir/skolasetning-2023

**Comment by Ronaldo-is-better** [comment]
hver er VÖLVAN?

**Comment by Ronaldo-is-better** [comment]
Ég vona að þú deyrð
